client/src/app.js

```diff
@@ -226,6 +227,7 @@ export default function main({ DOM, HTTP, route, storage, scanner: scan$, search
   // Currently visible view
   , view$ = O.merge(page$.mapTo(null)
                   , goHome$.mapTo('dashBoard')
+                  , goLanding$.mapTo('landing')
```

Comment:
`landing`/`lander` are pretty generic names that can mean different things, perhaps better to name this `api` or `apiServices` or something like this?

Reply:
updated to the `apiLanding`